Automattic/woocommerce-payments
# Add the new payment method logos to the connect page

Fixes #8808 

#### Changes proposed in this Pull Request
Create a new component that can be reused, to display the new payment method logos.
Design: https://www.figma.com/design/VLWDmeuIXQ7XAK4flyqTFO/Payments-Adoption-%26-Activation?node-id=2196-11373&t=7gXadrjGDmPqgpUP-0








#### Testing instructions
- go to [WcPay dev](http://localhost:8082/wp-admin/admin.php?page=wcpaydev)
- check `Force the WCPay plugin to act as disconnected from the WCPay Server` option and save the settings
- clear the cache
- re-onboard new account
- Go to Payments - the new component should be shown instead of the old payment method logos
<img width="710" alt="Screenshot 2024-06-18 at 13 07 51" src="https://github.com/Automattic/woocommerce-payments/assets/82826872/e0a51974-2125-484f-abac-1a60ad1f7942">






-------------------


**Post merge**
